### PR TITLE
[#89] Make it possible to add a required label

### PR DESCRIPTION
**Pull request by andreasknoepfle**
https://github.com/bitcrowd/bitstyles_phoenix/issues/89

### CODE REVIEW

File: lib/bitstyles_phoenix/component/form.ex [hunk 1]
```diff
@@ -547,10 +625,44 @@ defmodule BitstylesPhoenix.Component.Form do
...
 
   defp default_label(field), do: PhxForm.humanize(field)
+
+  defp required_label(%{required: value} = assigns)
+       when not is_nil(value) do
+    {module, function, opts} =
+      Application.get_env(:bitstyles_phoenix, :required_label, {
+        __MODULE__,
+        :default_required_label,
+        [foo: :bar]
```
Comment (hannahbit) [inline lib/bitstyles_phoenix/component/form.ex:642]: How about we make the default options an empty list?

File: lib/bitstyles_phoenix/component/form.ex [hunk 2]
```diff
@@ -547,10 +625,44 @@ defmodule BitstylesPhoenix.Component.Form do
...
+      <.required_label {assigns_to_attributes(assigns)} />
     <% end %>
     <.ui_errors form={@form} field={@field} />
     """
   end
 
   defp default_label(field), do: PhxForm.humanize(field)
+
+  defp required_label(%{required: value} = assigns)
+       when not is_nil(value) do
```
Comment (hannahbit) [inline lib/bitstyles_phoenix/component/form.ex:637]: Just wondering: 
Would it also make sense to check that the value is not `false`?

File: lib/bitstyles_phoenix/component/form.ex (hunk 1 above)
Comment (andreasknoepfle) [reply lib/bitstyles_phoenix/component/form.ex:642]: True, I set this for testing and then forgot 🙈

Comment (andreasknoepfle) [reply lib/bitstyles_phoenix/component/form.ex:642]:
```suggestion
        []
```

**Review by hannahbit — APPROVED** [review]
Awesome *
Looking forward to all those required labels  :)

**Pull request merged**